packages/qdrant-loader/tests/unit/webhooks/test_handlers.py

```
@@ -0,0 +1,68 @@
+class DummySettings:
+    pass
+
+
+def test_normalize_source_type_accepts_known_values():
+    assert normalize_source_type("Jira") == "jira"
+    assert normalize_source_type("confluence") == "confluence"
+    assert normalize_source_type("PublicDocs") == "publicdocs"
+
+
+def test_normalize_source_type_rejects_unknown_value():
+    with pytest.raises(ValueError):
```

Ruff (0.15.12)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)